### PR TITLE
Scatter plot component

### PR DESCRIPTION
**Pull request by knorris**
New component.

### CODE REVIEW

File: src/DefaultScatterPlot/DefaultScatterPlot.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import DataTable from '../DataTable'
+
+export class DefaultScatterPlot extends Component {
+  static propTypes = {
+    columns: PropTypes.array.isRequired,
+    data: PropTypes.array.isRequired,
+    series: PropTypes.array.isRequired,
+    size: PropTypes.string,
+    title: PropTypes.string.isRequired
+  };
```
Comment (jeffshaver) [inline src/DefaultScatterPlot/DefaultScatterPlot.jsx]: I updated babel on Friday, so we no longer need semi-colons after static properties.

**Comment by jeffshaver** [comment]
Just a note, I see a scatter plot issue (#17), but the commit message doesn't say it closes that issue

**Comment by knorris** [comment]
Sorry. I keep forgetting to add the issue to the commit message.

**Comment by jeffshaver** [comment]
Build failed due to an eslint/babel-eslint issues (eslint/eslint#5476). Still merging.

**Pull request merged**